lib/puppet_x/puppetlabs/scheduled_task/task.rb

```diff
@@ -372,6 +386,18 @@ def self.folder_path_from_task_path(task_path)
     path.empty? ? ROOT_FOLDER : path
   end
 
+  def self.create_folder(path)
+    service = task_service
+    begin
+      service.GetFolder(path)
```

Comment:
For ref : There is no API call for `exists`.  Just call get and wait for the error.

https://docs.microsoft.com/en-us/windows/desktop/api/taskschd/nn-taskschd-itaskfolder

Reply:
😭